Docs/source/theory/cold_fluid_model.rst

```diff
@@ -0,0 +1,135 @@
+.. _theory-cold-fluid-model:
+
+Cold Relativistic Fluid Model
+==========================
```

Comment:
Please balance the `=` with the title, rst is a bit picky.

Reply:
Added and opened a PR: https://github.com/ECP-WarpX/WarpX/pull/4330.